tests/e2e/default/contacts/contact-details.wdio-spec.js

```diff
@@ -32,13 +33,27 @@ describe('Contact details page', () => {
     const parent = placeFactory.place().build({ _id: 'dist1', type: 'district_hospital' });
     const user = userFactory.build({ username: 'offlineuser', roles: [role] });
     const patient = personFactory.build({ parent: { _id: user.place._id, parent: { _id: parent._id } } });
-    const report = reportFactory
-      .report()
-      .build(
+
+    const reports = Array
+      .from({ length: 60 })
+      .map(() => reportFactory.report().build(
         { form: 'pregnancy_danger_sign' },
-        { patient, submitter: user.contact, fields: { t_danger_signs_referral_follow_up: 'yes' },
+        {
+          patient,
+          submitter: user.contact,
+          fields: { t_danger_signs_referral_follow_up: 'yes' },
         }
-      );
+      ));
+
+    const pregnancyReport = [
```

Comment:
Unsure why this is built as an array only containing one element. Are more docs pushed later?

Reply:
No, there are no more docs pushed to it, that one element creates the pregnancy card that is tested later in the test.